docs/ko/docs/advanced/custom-response.md

```diff
@@ -0,0 +1,313 @@
+# 사용자 정의 응답 - HTML, Stream, 파일, 기타
+
+기본적으로, **FastAPI** 응답을 `JSONResponse`를 사용하여 반환합니다.
+
+이를 재정의 하려면 [응답을 직접 반환하기](response-directly.md){.internal-link target=_blank}에서 본 것처럼 `Response`를 직접 반환하면 됩니다.
+
+그러나 `Response` (또는 `JSONResponse`와 같은 하위 클래스)를 직접 반환하면, 데이터가 자동으로 변환되지 않으며 (심지어 `response_model`을 선언했더라도), 문서화가 자동으로 생성되지 않습니다(예를 들어, 생성된 OpenAPI의 일부로 HTTP 헤더 `Content-Type`에 특정 "미디어 타입"을 포함하는 경우).
+
+하지만 *경로 작업 데코레이터*에서 `response_class` 매개변수를 사용하여 원하는 `Response`(예: 모든 `Response` 하위 클래스)를 선언할 수도 있습니다.
+
+*경로 작업 함수*에서 반환하는 내용은 해당 `Response`안에 포함됩니다.
+
+그리고 만약 그 `Response`가 `JSONResponse`와 `UJSONResponse`의 경우 처럼 JSON 미디어 타입(`application/json`)을 가지고 있다면, *경로 작업 데코레이터*에서 선언한 Pydantic의 `response_model`을 사용해 자동으로 변환(및 필터링) 됩니다.
+
+/// note | 참고
+
+미디어 타입이 없는 응답 클래스를 사용하는 경우, FastAPI는 응답에 내용이 없을 것으로 예상하므로 생성된 OpenAPI 문서에서 응답 형식을 문서화하지 않습니다.
+
+///
+
+## `ORJSONResponse` 사용하기
+
+예를 들어, 성능을 극대화하려는 경우, <a href="https://github.com/ijl/orjson" class="external-link" target="_blank">orjson</a>을 설치하여 사용하고 응답을 `ORJSONResponse`로 설정 할수 있습니다.
+
+사용하고자 하는 `Response` 클래스(하위 클래스)를 가져와서 **경로 작동 테코레이터*에서 선언하세요.
+
+대규모 응답의 경우, 사전을 반환하는 것보다 `Response`를 반환하는 것이 훨씬 빠릅니다.
+
+이유는 기본적으로, FastAPI가 내부의 모든 항목을 검사하고 JSON으로 직렬화할 수 있는지 확인하기 때문입니다. 이는 사용자 안내서에서 설명된 [JSON 호환 가능 인코더](../tutorial/encoder.md){.internal-link target=_blank}를 사용하는 방식과 동일합니다. 이를 통해 데이터베이스 모델과 같은 **임의의 객체**를 반환할 수 있습니다.
+
+하지만 반환하는 내용이 **JSON으로 직렬화 가능**하다고 확신하는 경우, 해당 내용을 응답 클래스에 직접 전달할 수 있으며, FastAPI가 반환 내용을 `jsonable_encoder`를 통해 처리한 뒤 응답 클래스에 전달하는 추가적인 시간을 피할 수 있습니다.
+
+{* ../../docs_src/custom_response/tutorial001b.py hl[2,7] *}
+
+/// info | 정보
+
+`response_class` 매개변수는 응답의 "미디어 타입"을 정의하는 데에도 사용됩니다.
+
+이 경우, HTTP 헤더 `Content-Type`은 `application/json`으로 설정됩니다.
+
+그리고 이는 OpenAPI에 그대로 문서화됩니다.
+
+///
+
+/// tip | 팁
+
+`ORJSONResponse`는 FastAPI에서만 사용할 수 있고 Starlette에서는 사용할 수 없습니다.
+
+///
+
+## HTML 응답
+
+**FastAPI**에서 HTML 응답을 직접 반환하려면 `HTMLResponse`를 사용하세요.
+
+* `HTMLResponse`를 가져옵니다.
```

Comment:
Since "import" is a [Python-specific keyword](https://docs.python.org/3.13/reference/import.html), retaining it as "임포트" preserves its intended meaning. I recommend keeping it as is.

Other Korean translated documents, such as [body-fields](https://fastapi.tiangolo.com/ko/tutorial/body-fields/#field) and [body-nested-models](https://fastapi.tiangolo.com/ko/tutorial/body-nested-models/#typing-list), directly translate the term "import" as "임포트."

this review include line 25 and 55